test/libsolidity/syntaxTests/multiSource/import_alias.sol

```diff
@@ -0,0 +1,8 @@
+==== Source:s1.sol====
```

Comment:
https://github.com/ethereum/solidity/pull/10993#issuecomment-784109217

> Not sure why the tests are failing. Will try locally.

Maybe it's just that the file splitter expects whitespace here?

```suggestion
==== Source: s1.sol ====
```